docs/spectral_cube.rst

```diff
@@ -2,79 +2,141 @@
 Working with Spectral Cubes
 ###########################
 
-Spectral cubes can be read in using the
-`SpectralCube <https://spectral-cube.readthedocs.io/en/latest/>`_ package then
-collapsed using various methods to yield a Spectrum1D object.  A specific
-example of this workflow is demonstrated here.
+Spectral cubes can be read directly with :class:`~specutils.Spectrum1D`.
+A specific example of this is demonstrated here.
 
 Note that the workflow described here is for spectral cubes that are rectified
 such that one of the axes is entirely spectral and all the spaxels have the same
-``spectral_axis`` values - i.e., case 2 in
-:ref:`specutils-representation-overview`. For less-rectified cubes,
-pre-processing steps (not addressed by specutils at the time of this writing)
-will be necessary to rectify the cubes into that form.
+``spectral_axis`` values - i.e., case 2 in :ref:`specutils-representation-overview`.
```

Comment:
```suggestion
``spectral_axis`` values (i.e., case 2 in :ref:`specutils-representation-overview`).
```
I happened to notice the other day that the Astropy docs style guide says to put "i.e., ..." in parentheses.